src/main.rs

```diff
@@ -22,6 +22,13 @@ fn main() {
         .help("The status code of the previously run command")
         .takes_value(true);
 
+    let job_arg = Arg::with_name("job")
```

Comment:
I would consider naming this flag `jobs`, as we're expecting a count of potentially multiple jobs.

Reply:
will do

Reply:
done @matchai!